src/coreclr/tools/Common/JitInterface/CorInfoTypes.cs

```diff
@@ -881,6 +881,8 @@ public struct InlinedCallFrameInfo
         // Reverse PInvoke offsets
         public uint sizeOfReversePInvokeFrame;
 
+        public byte passMethodContextToReversePInvokeHelper;
```

Comment:
I think it would be more appropriate for this to be a `CORJIT_FLAG_`. The flag that controls callbacks for regular profiler enter/exit callouts is also `CORJIT_FLAG_`.